advisor-go/src/quixo/game/game.go

```diff
@@ -111,7 +111,7 @@ func isPlayerCube(cubeValue int, playerValue int) bool {
 	return cubeValue == 0 || cubeValue == playerValue
 }
 
-// Shift all value from xEnd to xStart in one row
+// Shift all values from xEnd to xStart in one row
```

Comment:
Don't hesitate to do git `squash` or git `fixup` for this kind of fix.

On my side, I made two shortcuts for these cases: `git fixup` and `git please`

```
// in ~/.gitconfig
[alias]
    commend = commit --amend --no-edit
    fixup = "!git add -p && git commend"
    please = push --force-with-lease
```

Reply:
Thank you for the tip :+1:

Reply:
If you use oh-my-zsh, you have a lot of shortcut, https://github.com/robbyrussell/oh-my-zsh/tree/master/plugins/git

I always use `gc!` (git commit -v --amend) in your case.